src/gluonts/mx/block/quantile_output.py

```diff
@@ -35,26 +36,31 @@ def __init__(
 
         Parameters
         ----------
-        quantiles
+        quantiles:
```

Comment:
The colon is actually not needed there

Reply:
Updated and removed the other colons from the comments here too!